src/gc/proxy.d

```diff
@@ -318,6 +321,20 @@ extern (C)
         return proxy.gc_inFinalizer();
     }
 
+    void gc_lock() nothrow @nogc
+    {
+        if( proxy is null )
+            return _gc.lock();
+        return proxy.gc_lock();
+    }
+
+    void gc_unlock() nothrow @nogc
+    {
+        if( proxy is null )
+            return _gc.unlock();
+        return proxy.gc_unlock();
+    }
```

Comment:
Existing proxies that don't have functions for these calls will cause a crash, right? Or is it required to implement these for compilation?